test/color/test_gray_via_conv.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+import kornia.color as color
+import kornia.color.via_conv as color_via_conv
```

Comment:
I think some other name like `covolutional` may be better, and maybe expose it in `color.__init__`